vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -94,7 +94,14 @@ func (e *StandardTxExecutor) CreateChainTx(tx *txs.CreateChainTx) error {
 	// If this proposal is committed and this node is a member of the subnet
 	// that validates the blockchain, create the blockchain
 	e.OnAccept = func() {
-		e.Config.CreateChain(txID, tx)
+		//TODO error
+		_ = e.ChainCreator.CreateChain(
+			txID,
+			tx.SubnetID,
+			tx.GenesisData,
+			tx.VMID,
+			tx.FxIDs,
+		)
```

Comment:
I'm not sure we want to error here. Currently we just log an error if creating a chain fails - rather than having the node FATAL... Bubbling the error here would cause the node to FATAL.